<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnotationFormatTableTab</class>
 <widget class="QWidget" name="AnnotationFormatTableTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: white;
}

QPushButton {
    background-color:rgb(154, 217, 234) ;
	color: white;
    min-width: 80px;
	min-height: 40px;
	border-style: solid;
	border-color: rgb(154, 217, 234);
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color:rgb(134, 189, 203) ;
	border-color: rgb(134, 189, 203);
	color: rgb(245, 245, 245);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget_annotformat_tablecontainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2" colspan="3">
       <widget class="QWidget" name="horizontalWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_annotformat_heading">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Specify Annotation Fromat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_help">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid white;
	color: grey;
}

QPushButton:hover {
    background-color: white;
    color: dark grey; 
	text-decoration: underline;
}

QPushButton:pressed {
    background-color: white;
}

QPushButton:focus {
    border: 1px solid white;
}

QPushButton::menu-indicator {
    image: none;
}</string>
           </property>
           <property name="text">
            <string>Help...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/images/Help_Icon.svg</normaloff>:/images/images/Help_Icon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QTableWidget" name="tableWidget_annotformats">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QAbstractItemView::indicator{
	width: 20px;
	height: 20px;
}

QAbstractItemView::indicator:checked {
    /* Adjust the appearance of the checked indicator */
    image: url(:/images/images/checked_icon.png) /* Specify a custom image if desired */
}

QAbstractItemView::indicator:unchecked {
    /* Adjust the appearance of the unchecked indicator */
    image: url(:/images/images/unchecked_icon.png); /* Specify a custom image if desired */
}


QTableWidget {
    font-size: 12pt; /* Set the font size */
    font-weight: bold; /* Set the font weight */
}

QTableWidget::item:selected {
    background-color: rgba(50, 50, 225, 0.1); /* Change the background color and opacity */
	color:black;
}

QTableWidget::item:focus {
    selection-color: blue; /* Hide the dotted selection border */
}

QAbstractItemView::indicator:hover {
    /* Adjust the appearance of the hover indicator */
    background-color: lightgray; /* Example: change background color when hovering */
}

QAbstractItemView::indicator:checked:hover {
    /* Adjust the appearance of the checked checkbox when hovering */
    background-color: rgb(255, 166, 167); /* Example: change background color of checked checkbox when hovering */

}


QAbstractItemView::indicator:unchecked:hover {
    /* Adjust the appearance of the unchecked checkbox when hovering */
    background-color: lightgreen; /* Example: change background color of unchecked checkbox when hovering */
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>600</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>40</number>
        </attribute>
        <column>
         <property name="text">
          <string>Annotation format</string>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <italic>true</italic>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Regular Expression</string>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <italic>true</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
        </column>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_annotformat_description">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Overview of your specified annotation formats for the current project. Each annotation in your pre-annotated corpus that matches the pattern of one of the specified annotation format will be detected and further processed. In this menu, annotation formats can be added, edited, and deleted so that CorpusCompass can detect all annotations for your individual annotation style.</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QWidget" name="widget_addedit" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_annotformat_addnew">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Add annotation format</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/images/Plus_Icon.svg</normaloff>:/images/images/Plus_Icon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_annotformat_edit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Edit annotation format</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/images/edit_icon_white.png</normaloff>:/images/images/edit_icon_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_annotformat_buttoncontainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_annotformat_cancel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: white;
	background-color: lightgrey;
	border-width: 2px;
	border-radius: 10px;
	border-color: grey;

}

QPushButton {
	color: black;
	background-color: white;
	border-style: solid;
	border-color: black;
	border-width: 2px;
	border-radius: 10px;
	min-height: 80px;
}

QPushButton:pressed {
	background-color: rgb(120, 120, 120);
}

QPushButton:disabled{
  background-color: rgb(121, 121, 121);
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_annotformat_save">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: white;
	background-color: rgb(0, 170, 0);
	border-style: solid;
	border-color: rgb(0, 125, 0);
	border-width: 2px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: rgb(0, 134, 0);
}

QPushButton:disabled{
  background-color: rgb(121, 121, 121);
}</string>
        </property>
        <property name="text">
         <string>Save changes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
